### PR TITLE
Requesting a pull to mozilla:master from k88hudson:t1811

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/8ca31c4257d0dd8/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/8ca31c4257d0dd8/output.txt

Total script time: 0.80 mins

Server is running on http://pr753.butyr.org/templates/basic/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/7598ec365dddddf/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/7598ec365dddddf/output.txt

Total script time: 0.87 mins

File: css/editors.less [hunk 1]
```diff
@@ -1,222 +1,258 @@
...
+          margin-right: 0;
+          border: none;
+          box-shadow: none;
+          border-radius: 0;
+
+          &.butter-active {
+            background: #41bb92;
+            text-shadow: 0 1px 1px #276f57;
+          }
+          >i {
```
Comment (jbuck) [inline css/editors.less]: spacing

File: css/editors.less [hunk 2]
```diff
@@ -1,222 +1,258 @@
...
+            margin-right: 5px;
+          }
+        }
+
+      }
+    }
+  } // .butter-editor-header
+
+ .butter-editor-close-btn {
+  position: fixed;
```
Comment (jbuck) [inline css/editors.less]: indent is off

File: css/editors.less [hunk 3]
```diff
@@ -1,222 +1,258 @@
...
-        content: "";
-        display: table;
-        clear: both;
-      }
 
-      > li {
-        float: left;
-        padding: 10px 0 10px 0;
+  // Spacing 
+ .butter-editor-spacing {
```
Comment (jbuck) [inline css/editors.less]: spacing

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/ae7af451cb809f5/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/ae7af451cb809f5/output.txt

Total script time: 0.88 mins

Server is running on http://pr753.butyr.org/templates/basic/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/08413c4525525e4/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/08413c4525525e4/output.txt

Total script time: 0.93 mins

Server is running on http://pr753.butyr.org/templates/basic/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/d7e53d23d5f8049/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/d7e53d23d5f8049/output.txt

Total script time: 1.14 mins

Server is running on http://pr753.butyr.org/templates/basic/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/0a2f3e20ab86b53/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/0a2f3e20ab86b53/output.txt

Total script time: 1.04 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/2fc68f11552b7ca/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/2fc68f11552b7ca/output.txt

Total script time: 0.67 mins

Server is running on http://pr753.butyr.org/templates/basic/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/c8ee3bbd7643521/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/c8ee3bbd7643521/output.txt

Total script time: 1.02 mins

Server is running on http://pr753.butyr.org/templates/basic/
Run the Test Suite: http://pr753.butyr.org/test/

File: src/editor/editor.js [hunk 4]
```diff
@@ -88,6 +88,7 @@ define( [ "util/lang", "util/xhr",
           completeLayout,
           compiledLayout;
 
+      console.log( __editors);
```
Comment (mjschranz) [inline src/editor/editor.js]: A wild console.log appears!

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/86af331249087af/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/86af331249087af/output.txt

Total script time: 0.67 mins

Server failed to start

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/5cf32258a105d38/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/5cf32258a105d38/output.txt

Total script time: 1.06 mins

**Comment by jbuck** [comment]
Staged on master: https://github.com/mozilla/butter/commit/3da05faaa24fa7dffccb02bcbb06b55c315f146b

**Pull request closed without merging**